app/assets/stylesheets/components/_preview-gem-cards.scss

```diff
@@ -0,0 +1,84 @@
+/* ===================================================== *\
+   preview-gem-cards
+\* ===================================================== */
+
+.preview-gem-cards {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+
+.preview-gem-card {
+  border: 1px solid darken($light-blue, 8%);
+  border-radius: 4px;
+  margin-bottom: 10px;
+  flex: 0 1 auto;
+}
+
+.preview-gem-card-header {
+  border-bottom: 1px solid darken($light-blue, 8%);
+  background-color: $light-blue;
+  color: $ocean-blue;
+  border-top-left-radius: 4px;
+  border-top-right-radius: 4px;
+  display: flex;
+  align-items: center;
+
+  .eui-btn--blue {
+    background-color: lighten($midnight-blue, 4%);
```

Comment:
seems obvious, but might be helpful for future devs to add a comment that this is using the eui class, but the actual color will be different